src/phases/filesystem/FlushingTemplate.yml

```diff
@@ -0,0 +1,1656 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This file defines a template to use to stress checkpointing, journal flush and dirty memory. The test itself is
+  simple:
+    - Run a single CrudActor targeting:
+      - test database, configurable via the Database parameter.
+      - Collection0 collection, configurable via the Collection parameter.
```

Comment:
This doesn't look like it's set by parameter. Is the comment wrong or the code below?

Reply:
I see now that you've specificed the parameter much further down. That's mildly confusing. Can we set them all in one place?